src/components/tree-view/tree-view.tsx

```diff
@@ -56,6 +56,12 @@ export type ParentsMap = Map<TreeItemData, TreeItemData | undefined>;
 
 const itemIdPrefix = 'TREE_ITEM';
 
+// This function is not perfect but for now this is
+// a solution that will be changed later
+function getFocusedItemKey(item: TreeItemData) {
```

Comment:
why do you need it? you call it twice, can't you just use item.label where you call it?

Reply:
it is a bit YAGNI indeed